### PR TITLE
feat: support details view for ERC721 NFTs

### LINKED ISSUE #1568: ️🛠 - Support detailed view ERC721 NFTs
### Task description

With the ERC721 NFTs being loaded into appropriate stores, we can actually display them on the UI.

### Requirements

_TBD_

### PR DESCRIPTION
**Pull request by maxwellmattryan**
## Summary

> Please summarize your changes, describing **what** they are and **why** they were made.

...

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [x] Linux
    -   [ ] Windows

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [x] I have followed the contribution guidelines for this project
-   [x] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by nicole-obrien** [comment]
Commented some issues in discord, will review the code tomorrow

File: packages/desktop/components/menus/CollectibleDetailsMenu.svelte [hunk 1]
```diff
@@ -12,28 +12,11 @@
     export let menu: Menu = undefined
     export let nft: INft
 
-    $: url = nft?.metadata?.uri && composeUrl(nft.metadata.uri)
+    $: url = nft?.composedUrl ?? composeUrlFromNftUri(nft?.metadata?.uri)
     $: isLocked = isNftLocked(nft)
+    $: isBurnDisabled = isLocked || !isIrc27Nft(nft)
```
Comment (Tuditi) [inline packages/desktop/components/menus/CollectibleDetailsMenu.svelte:17]: We could also implement NFT burning functionality by sending to the 0x0 address (null address) or `0x000000000000000000000000000000000000dEaD` (burn address).

**Review by Tuditi — APPROVED** [review]
I pushed some code, so make sure to review it before you merge this one.

File: packages/desktop/components/menus/CollectibleDetailsMenu.svelte (hunk 1 above)
Comment (maxwellmattryan) [reply packages/desktop/components/menus/CollectibleDetailsMenu.svelte:17]: Ah true, should be easy to do once we get send flow impl. working

File: packages/shared/src/lib/core/nfts/enums/nft-standard.enum.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
 export enum NftStandard {
     Irc27 = 'IRC27',
-    Erc721 = 'ERC721',
+    Erc721 = 'ERC-721',
```
Comment (maxwellmattryan) [inline packages/shared/src/lib/core/nfts/enums/nft-standard.enum.ts:3]: @Tuditi why are we making this change?

File: packages/shared/src/lib/core/token/enums/token-standard.enum.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 export enum TokenStandard {
     BaseToken = 'BASE_TOKEN',
     Irc30 = 'IRC30',
-    Erc20 = 'ERC20',
+    Erc20 = 'ERC-20',
```
Comment (maxwellmattryan) [inline packages/shared/src/lib/core/token/enums/token-standard.enum.ts:4]: Same as above ^^^

File: packages/shared/src/lib/core/nfts/enums/nft-standard.enum.ts (hunk 2 above)
Comment (Tuditi) [reply packages/shared/src/lib/core/nfts/enums/nft-standard.enum.ts:3]: To follow the specification.
TIP: 
image
EIP:
image

Comment (Tuditi) [reply packages/shared/src/lib/core/nfts/enums/nft-standard.enum.ts:3]: It is specified in this format in the relevant TIP/EIPs

I'm easy on this. Feel free to revert the changes if you feel strongly about it.

The fact that it cleans up the code, does give me the hunch that it's the right thing to do.

**Pull request merged**